### DISCUSSION TITLE
Remove or change default binding for toggle left dock?

### DISCUSSION THREAD

**Discussion by mskri** [zed-industries/zed#13490] [Feature Requests]
Using the latest version (0.140.5). I'm trying to remove the default binding `cmd-b` for toggling the left dock. I would prefer to use the shortcut for go to implementation instead. However, I haven't figured out a working configuration to change the toggle left dock shortcut.

I've tried this one but it doesn't seem to have any effect. `cmd-b` still toggles the left dock and `cmd-g` does nothing.

```json
[
 {
    "context": "Editor",
    "bindings": {
      "cmd-b": "editor::GoToImplementation"
    }
},
 {
    "context": "Workspace",
    "bindings": {
      "cmd-g": "workspace::ToggleLeftDock",
    }
  }
]
```

What am I missing? 🤔  I would like to achieve binding `cmd-b` to "editor::GoToImplementation" and move "workspace:ToggleLeftDock" to `cmd-g`.

Trying to change the default bindings for "workspace::ToggleRightDock" and "workspace::ToggleBottomDock" doesn't work either.

Edit. downgrading to 0.139.3 seems to make the custom shortcuts work again.

**Comment by mskri** [comment]
Further testing shows that none of the custom shortcuts from keymaps work 🤔

**Reply by wakelin-g** [reply]
Seems to be working for me, at least on `0.141.3`.

I think this is probably an issue of contexts. For example, with my cursor inside an editor pane in normal vim mode,

```json
[
  {
    "bindings": {
      "ctrl-b": "workspace::ToggleLeftDock"
    }
  }
]
```

works, whereas

```json
[
  {
    "context": "Workspace",
    "bindings": {
      "ctrl-b": "workspace::ToggleLeftDock"
    }
  }
]
```

does not.
